### DISCUSSION TITLE
Video Lagging

### DISCUSSION THREAD

**Discussion by jcostel2** [biigle/core#536] [Q&A]
Hi there, 
The past two days i've been experiencing a lot of video lagging while annotating in biigle. Is there anything that I can do to solve this?

Thanks!

**Comment by mzur** [comment]
Is the video playing (fluidly) and pausing frequently? Then maybe your internet connection is currently slow.

Or does the video stutter constantly? Then the video resolution is probably quite large and your computer struggles to handle the playback. The video player of BIIGLE is not very optimized, currently, and can run into performance issues for very large videos or videos with lots of annotations.

**Reply by jcostel2** [reply]
The video is playing and then pauses frequently. It does have a lot of annotations, but it is happening with videos that haven't been annotated yet as well.

**Reply by mzur** [reply]
Then it's probably your internet connection again. Since you uploaded the videos to the BIIGLE server, they have to be loaded from there now. You can try with your older volume (with the remote videos) and see if video playback is better there.
